move/interchain_token_service/sources/interchain_token_service.move

```diff
@@ -331,6 +331,8 @@ module interchain_token_service::interchain_token_service {
         );
     }
 
+    /// This can be used bye the person who added the treasury cap to reclaim mint/burn permission from ITS.
```

Comment:
```suggestion
    /// This can be used by the `deployer` who added the treasury cap to reclaim mint/burn permission from ITS for that `token_id`.
```